en_US.ini

```diff
@@ -135,7 +135,7 @@ Log Console = &Log Console
 Memory View... = Memory &View...
 More Settings... = &More Settings...
 Nearest = &Nearest
-Non-Buffered Rendering = &Non-Buffered Rendering (speedhack)
+Non-Buffered Rendering = &Discard Effects (Non-Buffered, speedup)
```

Comment:
You need something that says in two words or something that "it does not process effects" or "effect process skip". Discarding sounds more like it does the job and then throws it out the window. :p Dunno I'm puzzled :/